rpm/stackdriver-agent.spec

```diff
@@ -234,7 +235,7 @@ popd
 %{__make} install DESTDIR="%{buildroot}"
 
 %{__install} -Dp -m0755 %{SOURCE200} %{buildroot}/%{_initddir}/stackdriver-agent
-%{__install} -Dp -m0644 %{SOURCE201} %{buildroot}%{_sysconfdir}/collectd.conf
+%{__install} -Dp -m0644 %{SOURCE201} %{buildroot}/%{_confdir}/collectd.conf
```

Comment:
Doesn't `%{_confdir}` already start with a `/`?

Reply:
So do `_initdir` and `_sysconfdir`. I stuck to the format in this file.